core/src/cost_model.rs

```diff
@@ -147,7 +147,7 @@ impl CostModel {
     pub fn upsert_instruction_cost(
         &mut self,
         program_key: &Pubkey,
-        cost: &u64,
+        cost: u64,
```

Comment:
Makes perfect sense - no need of ref on u64. Did that just for cosmetic reason - so all parameters are in same "fashion". Bit silly :) All for removing ref for better practice.